source/includes/get-started/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.11 FATAL_ERROR)
```

Comment:
The `FATAL_ERROR` keyword can be dropped, as it has no effect on any recent CMake version.